SwiftUIDZ/ContentView.swift

```diff
@@ -1,21 +1,209 @@
+        static let wifiStatuses = ["Подключен", "Не подключен"]
+        static let setUpStatuses = ["Вкл.", "Выкл."]
+        static let mobileStatuses = ["Yota", "MTC"]
+        static let navigationTitleText = "Настройки"
+        static let yinYangImageName = "yinYang"
+        static let aviaImageName = "avia"
+        static let wifiImageName = "wifi"
+        static let bluetoothImageName = "bluetooth"
+        static let mobileImageName = "mobile"
+        static let modemImageName = "modem"
+        static let vpnImageName = "vpn"
+        static let chevronForwardImageName = "chevron.forward"
+        static let userName = "Alexander"
+        static let profileTitleText = "Apple ID, iCloud, контент и покупки"
+        static let recomendationTitleText = "Предложения Apple ID"
+        static let updateTitleText = "iOS 16.2: уже доступно"
+        static let aviaTitleText = "Авиарежим"
+        static let wifiTitleText = "Wi-Fi"
+        static let bluetoothTitleText = "Bluetooth"
+        static let mobileTitleText = "Сотовая связь"
+        static let modemTitleText = "Режим модема"
+        static let vpnTitleText = "VPN"
+        static let recomendationNotificationCount = "3"
+        static let updateNotificationCount = "1"
+        static let commonCornerRadius: CGFloat = 10
+        static let commonSizeNumber: CGFloat = 30
+        static let profileSizeNumber: CGFloat = 60
+        static let userFontSizeNumber: CGFloat = 25
+        static let userDescriptionFontSizeNumber: CGFloat = 15
+        static let zeroNumber = 0
+    }
+    
+    @State private var isOnAvia = false
+    @State private var isOnVPN = false
+    @State private var wifiSelection = Constants.zeroNumber
+    @State private var bluetoothSelection = Constants.zeroNumber
+    @State private var mobileSelection = Constants.zeroNumber
+    @State private var modemSelection = Constants.zeroNumber
+    
+    private var wifiStatuses = Constants.wifiStatuses
+    private var setUpStatuses = Constants.setUpStatuses
+    private var mobileStatuses = Constants.mobileStatuses
+
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        NavigationView {
+            Form {
+                Section {
+                    setupProfileView()
+                    setupRecomendarionView()
+                }
+                Section {
+                    setupUpdateView()
+                }
+                Section {
+                    setupAviaView()
+                    setupWiFiView()
+                    setupBluetoothView()
+                    setupMobileView()
+                    setupModemView()
+                    setupVpnView()
+                }
+            }
+            .navigationTitle(Constants.navigationTitleText)
+        }
+    }
+    
+    private func setupProfileView() -> some View {
```

Comment:
```suggestion
    private var profileView: some View {
```
Если нет параметров значит выносим в вычисляемое свойство а не в метод